src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```
@@ -162,36 +162,81 @@ admin_products_v2_duplicate:
   requirements:
     productId: \d+
 
-admin_products_v2_toggle_status:
-  path: /{productId}/toggle_status/{shopId}
+admin_products_v2_duplicate_shop:
+  path: /{productId}/duplicate/shop/{shopId}
   methods: [ POST ]
   defaults:
-    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::toggleStatusAction
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::duplicateShopAction
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:duplicate
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+    shopId: \d+
+
+admin_products_v2_duplicate_shop_group:
+  path: /{productId}/duplicate/shop-group/{shopGroupId}
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::duplicateShopGroupAction
+    _legacy_controller: AdminProducts
+    _legacy_link:
+      - AdminProducts:duplicate
+    _legacy_parameters:
+      id_product: productId
+  requirements:
+    productId: \d+
+    shopGroupId: \d+
+
+admin_products_v2_toggle_status_for_shop:
+  path: /{productId}/toggle_status/shop/{shopId}
```

not sure about this one but smth like:
```suggestion
  path: /{productId}/toggle-status-shop/{shopId}
```